test/e2e/bootstrapper_test.go

```diff
@@ -110,3 +143,18 @@ func getSvcState(svcName string) (svc.State, error) {
 	}
 	return status.State, nil
 }
+
+// removeFileIfExists removes the file given by 'path', and will not throw an error if it does not exist
+func removeFileIfExists(t *testing.T, path string) {
+	err := os.Remove(path)
+	if err != nil {
```

Comment:
This should be a `require.NoError(t, err)`

Reply:
There are cases where an error is fine, such as if the file doesnt exist

Reply:
Are you saying that `Remove()` throws errors even if the file has been removed?

Reply:
I don't think that is the case: https://golang.org/src/os/file_unix.go?s=9421:9451#L301

Reply:
Thats what I was saying, wouldnt it throw an ENOENT error?

Reply:
You mean `ENOTDIR`. That can only happen in the case of of a directory.

Reply:
https://linux.die.net/man/2/unlink
`ENOENT` is there:
`    A component in pathname does not exist or is a dangling symbolic link, or pathname is empty. `

Reply:
In windows it would be an `ERROR_FILE_NOT_FOUND`

Reply:
Understood. However it would still be `require` vs an `assert` within the `if err != nil`